airflow/models/taskinstance.py

```diff
@@ -2714,18 +2714,29 @@ def signal_handler(signum, frame):
                 previous_state=TaskInstanceState.QUEUED, task_instance=self, session=session
             )
 
-            # Execute the task
+            def _render_map_index(context: Context, *, jinja_env: jinja2.Environment | None) -> str | None:
+                """Render named map index if the DAG author defined map_index_template at the task level."""
+                if jinja_env is None or (template := context.get("map_index_template")) is None:
+                    return None
+                rendered_map_index = jinja_env.from_string(template).render(context)
+                log.info("Map index rendered as %s", rendered_map_index)
```

Comment:
Is this log useful?

Reply:
Yeah, worth removing or changing it to debug level (if we think it could be useful in debugging)

Reply:
@RNHTTR thoughts? (Since you actually used this and debugged errors with it in practice)

Reply:
I think it was useful before I hooked up the UI, but it should be fine to remove.

Reply:
Looks like it got changed to debug by now, which is fine.